tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -1139,15 +1146,20 @@ def create_and_validate_custom_host(node_roles, random_cluster_name=False,
 def create_custom_host_from_nodes(nodes, node_roles,
                                   random_cluster_name=False, windows=False,
                                   windows_flannel_backend='vxlan',
-                                  version=K8S_VERSION):
+                                  version=K8S_VERSION, 
+                                  cloudprovider=CLOUDPROVIDER_ENABLED):
     client = get_user_client()
     cluster_name = random_name() if random_cluster_name \
         else evaluate_clustername()
 
     if windows:
-        if windows_flannel_backend == "host-gw":
+        if windows_flannel_backend == "host-gw" and cloudprovider:
```

Comment:
Isn't `cloudprovider` always be boolean `True` here ? `CLOUDPROVIDER_ENABLED` is a string and non-empty will always return True.

Reply:
i think you are right. Changing it and testing it set to false. I checked for "CLOUDPROVIDER_ENABLED" = True earlier.

Reply:
Works now with the latest changes. Tested it locally with cloudprovider = True